### PR TITLE
bug in the sanity check after maxfilters.

### PR DESCRIPTION
**Pull request by crsegerie**
The sanity check comparing the rank of the experimental data and the rank of the empty-room after the maxfilter did not use the maxfiltered matrices

### Before merging …

-  [x] Changelog has been updated (`docs/source/changes.md`)

### CODE REVIEW

**Comment by hoechenberger** [comment]
Thanks @crsegerie!

I've just merged #335, which should solve the early CI errors we're seeing here. Can you pull `main` into your branch?

File: docs/source/changes.md [hunk 1]
```diff
@@ -65,3 +65,5 @@ authors:
 - The summary report didn't use the cleaned epochs for showing the effects of
   ICA.
   ({{ gh(320) }} by {{ authors.hoechenberger }})
+- The sanity check comparing the rank of the experimental data and the rank of the empty-room after the maxfilter did not use the maxfiltered matrices
```
Comment (agramfort) [inline docs/source/changes.md:68]:
```suggestion
- The sanity check comparing the rank of the experimental data and the rank of the empty-room after the maxfilter did not use the maxfiltered data
```

**Review by agramfort — APPROVED** [review]
just a nitpick

**Review by agramfort — APPROVED** [review]
@hoechenberger merge if you think it will break CI effectively

**Comment by hoechenberger** [comment]
> @hoechenberger merge if you think it will break CI effectively

😅

File: docs/source/changes.md [hunk 2]
```diff
@@ -65,3 +65,5 @@ authors:
 - The summary report didn't use the cleaned epochs for showing the effects of
   ICA.
   ({{ gh(320) }} by {{ authors.hoechenberger }})
+- The sanity check comparing the rank of the experimental data and the rank of the empty-room after the maxfilter did not use the maxfiltered data
```
Comment (hoechenberger) [inline docs/source/changes.md:68]:
```suggestion
- The sanity check comparing the rank of the experimental data and the rank of the empty-room after Maxwell-filtering did not use the maxfiltered data
```

File: docs/source/changes.md [hunk 3]
```diff
@@ -65,3 +65,5 @@ authors:
...
+  (by {{ authors.agramfort }}, {{ authors.hoechenberger }} and {{ crsegerie }}).
```
Comment (hoechenberger) [inline docs/source/changes.md:69]:
```suggestion
  (by {{ authors.agramfort }}, {{ authors.hoechenberger }}, and {{ crsegerie }}).
```

**Review by hoechenberger — APPROVED** [review]
nitpicks

**Comment by hoechenberger** [comment]
Thanks, @crsegerie!

**Comment by agramfort** [comment]
👍 @crsegerie 🙏

**Pull request merged**